controllers/users/controllers.js

```diff
@@ -94,3 +103,46 @@ export const avatars = async (req, res, next) => {
     next(error);
   }
 };
+
+export const verify = async (req, res, next) => {
+  try {
+    const user = await User.findOne({
+      verificationToken: req.params.verificationToken,
+    });
+    if (!user) {
+      return res.status(404).json({ message: "User not found" });
+    }
+    user.verificationToken = "done";
+    // nie moglem ustawić verificationToken na null, bo w modelu jest required
+    // nie wiem czy to błąd zadania, czy ja nie rozumiem jak to obejść
+    user.verify = true;
```

Comment:
To też robi robotę. Tylko potem musisz się upewnić że sprawdzasz czy użytkownik został zweryfikowany poprzez verify.